test/edu/stanford/dlss/wowza/TestValidationMethods.java

```diff
@@ -0,0 +1,184 @@
+package edu.stanford.dlss.wowza;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+import org.junit.*;
+
+import org.apache.log4j.*;
+
+import java.io.ByteArrayOutputStream;
+
+public class TestValidationMethods
+{
+    SulWowza testModule;
+    static String stacksToken = "encryptedStacksMediaToken";
```

Comment:
i'd be inclined to make this look more like a java style constant (`final` and named in all caps).  but am 👌 leaving it as-is for expediency's sake.